aws/table_aws_route53_vpc_association_authorization.go

```diff
@@ -0,0 +1,134 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/route53"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsRoute53VPCAssociationAuthorization(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_route53_vpc_association_authorization",
+		Description: "AWS Route53 VPC Association Authorization",
+		List: &plugin.ListConfig{
+			ParentHydrate: listHostedZones,
+			Hydrate:       listVPCAssociationAuthorization,
+			Tags:          map[string]string{"service": "route53", "action": "ListVPCAssociationAuthorizations"},
```

Comment:
Could you please add the `hosted_zone_id` as the optional key qulas here, which will help us reduce the API call and the query timing? If a user wants the rows for a particular Hosted Zone we should not make the API call for other Hosted Zones.